subsys/bluetooth/host/shell/iso.c

```diff
@@ -845,9 +846,7 @@ static int cmd_big_sync(const struct shell *sh, size_t argc, char *argv[])
 				return -ENOEXEC;
 			}
 
-			if (!IN_RANGE(mse,
-				      BT_ISO_SYNC_MSE_MIN,
-				      BT_ISO_SYNC_MSE_MAX)) {
+			if (mse != 0 && !IN_RANGE(mse, BT_ISO_SYNC_MSE_MIN, BT_ISO_SYNC_MSE_MAX)) {
```

Comment:
Suggest to add a `BT_ISO_SYNC_MSE_ANY` value in `iso.h` and do
```suggestion
			if (mse != BT_ISO_SYNC_MSE_ANY && !IN_RANGE(mse, BT_ISO_SYNC_MSE_MIN, BT_ISO_SYNC_MSE_MAX)) {
```

Reply:
Oh :D